src/openforms/js/components/form/edit/options.js

```diff
@@ -54,6 +54,14 @@ const SHOW_IN_PDF = {
   defaultValue: true,
 };
 
+const AUTOCOMPLETE = {
+  type: 'checkbox',
+  key: 'autocomplete',
+  label: 'Autocomplete',
+  tooltip: 'Display options to fill in the field, based on earlier typed values.',
+  defaultValue: true,
```

Comment:
I don't see a/the default value in the upstream configuration: https://github.com/formio/formio.js/blob/master/src/components/textfield/editForm/TextField.edit.display.js#L155

I'd reckon we should mostly copy this configuration (but use a radio or dropdown instead for the values). For the full list of values, see: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#values ... maybe it's better to leave this as textfield after all :thinking:

Reply:
@sergei-maertens and @joeribekker yes: I guess the idea should be to eventually make a Readme/manual where the user can enter all possibles values for Autocomplete, ranging from on/off to "family-name" etc.

Reply:
users will not do that because it slows them down and distracts them from Getting Shit Done. Optional fields are usually left untouched except by those people that have a passion for getting these sort of things right, but that's only a small subset of people.

Reply:
ToDo:
- make autocomplete dropdown with at least these values: https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css-textbox--page
- use Presets (remains a Textfield)
- move 'Postcode' form component to 'depracated' + migrate